index.markdown

```diff
@@ -1123,6 +1132,18 @@ Pretty much anything on Castro Street.
 
 Everything here is pretty much centered around University Ave.
 
+#### San Mateo
+
+Restaurant heaven. Did a 8 month co-op here for a startup. 4 by 4 block of 
```

Comment:
Can you remove "Did a 8 month co-op here for a startup. 4 by 4 block of just pure restaurants (all within 2-8 minutes walk of the caltrain station) my favorites"?